tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/statements/vardeclr/ModulePublicVariableTest.java

```diff
@@ -54,6 +54,7 @@ public void testModulePublicVariableAccess(String functionName) {
                 "testPublicVisibilityInComplexVar",
                 "testPublicWithIsolatedFuncType",
                 "testPublicWithIsolatedObjectType",
+                "testPublicFunctionWithRecordType"
```

Comment:
Is this the correct place for this test? There are no module-level variables in the test.

IMO, we don't need a test with modules here. We can add a simple test in a single bal file.

Reply:
Maybe a new `org.ballerinalang.test.access.AccessLevelsTest` at the same level as `org.ballerinalang.test.access.AccessLevelsNegativeTest`.

Reply:
Added because Record type is defined in module level. Will Change it